ManagerDTO.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using System.Reflection;
+using System.Text;
+using AirportTickets.Booking;
+using AirportTickets.Flight;
+using static AirportTickets.Flight.FlightDTO;
+
+namespace AirportTickets
+{
+    public class ManagerDTO
+    {
+        public string Name { get; set; }
+
+        public ManagerDTO()
+        {
+
+        }
+
+        public List<ReservationDTO> FilterByFlight(FlightDTO flight)
+        {
+            return
+                BookingDTO.RetreiveBookings()
+                .Where(item => item.Flight == flight)
+                .ToList();
+        }
+
+        public string PropertyAttributes()
```

Comment:
The methods here are not really the responsibility of ManagerDTO. DTOs usually just to hold data, and have simple methods that run on the class properties. 
`PropertyAttributes` is more a responsibility for flights, I would create a different service/extension class to deal with the operations on flights. Follow this logic for the other methods here.